### PR TITLE
robot field commands → dispenser actuation subsystem prototype

### PR DESCRIPTION
**Pull request by steakhouserodriguez**
uses a stand-in piemos script to trigger simulated robot field commands. The interface should be drop-in once piemos implements.

rest of the system is simple but complete.

### CODE REVIEW

**Comment by steakhouserodriguez** [comment]
I would also like to hear @zentner-kyle's comments.

File: src/dispenser_field.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python2.7
+
+"""
+This node controls the real field elements
+"""
+
+import lcm
+import forseti2 as fs2
+import IPython.display
```
Comment (chunjiang) [inline src/dispenser_field.py]: delete delete delete delete

Comment (steakhouserodriguez) [inline src/dispenser_field.py]: bad

File: src/dispenser_vis.py [hunk 2]
```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python2.7
+
+"""
+This node visualizes what the field dispensers should be doing.
+"""
+
+import lcm
+import forseti2 as fs2
+import IPython.display
```
Comment (chunjiang) [inline src/dispenser_vis.py]: delete delete delete delete

**Comment by chunjiang** [comment]
I have fixed most of these things. Will commit once tested on octy.

Still need to add dependencies to readme, LCM channel descriptions to readme, and make LCM network set-able at one location

**Pull request merged**